src/Parser/RetributionPaladin/Modules/Features/CastEfficiency.js

```diff
@@ -25,7 +27,8 @@ class CastEfficiency extends CoreCastEfficiency {
       getCooldown: haste => 30,
       isActive: combatant => combatant.hasShoulder(ITEMS.ASHES_TO_DUST.id),
```

Comment:
(For some reason I can't add the comment to the line I want, which is Wake of Ashes above)

Shouldn't the "regular" Wake of Ashes be constrained with `isActive: combatant => !combatant.hasShoulder(ITEMS.ASHES_TO_DUST.id)`? This way, if the player has those shoulders, it would show Wake of Ashes twice in the Cast Efficiency. These should be mutually exclusive, either you have those shoulders or not.

Reply:
This is paladin stuff, it somehow made it's way into this, and also part of the reason, I think this one should just be closed (I fucked something up in git somewhere), and as such has nothing to do with this PR 👎 
The t20 one does not include anything outside of hunter stuff 😦

Reply:
Well, it looks like the commits you made in this PR managed to stay even in the newer T20 ones, so if you want to, I can close this PR by your request (assuming you fixed the issues Zero wanted you to fix) and review the new one.

Reply:
👍 closing it myself